src/FFMpeg/Filters/Video/ResizeFilter.php

```diff
@@ -98,8 +98,10 @@ public function apply(Video $video, VideoInterface $format)
         if (null !== $dimensions) {
             $dimensions = $this->getComputedDimensions($dimensions, $format->getModulus());
 
-            $commands[] = '-s';
-            $commands[] = $dimensions->getWidth() . 'x' . $dimensions->getHeight();
+            // Using Filter to have ordering
+            $commands[] = '-vf';
+            $commands[] = '[in]scale=' . $dimensions->getWidth() . ':' . $dimensions->getHeight() . ' [out]';
```

Comment:
Reference: http://stackoverflow.com/questions/10934420/ffmpeg-how-to-scale-a-video-then-apply-a-watermark